scripts/run_tests.py

```diff
@@ -489,36 +489,33 @@
     "cwl_dynamic_initial_workdir",
     "cwl_expressionLib",
     "cwl_format",
-    # "cwl_format_url", # APPS-961 Could not load extension schema https
+    "cwl_format_url", # APPS-1111
```

Comment:
Does the APPS-1111 comment mean this one is still not working?

Reply:
Thanks @r-i-v-a ! I actually fixed the test code but the issue itself in the compiler is not resolved yet. I have reverted my changes to the test and I will bring it back once the bug is fixed.